### DISCUSSION TITLE
Server hadle multiple subscriptions with same nodes

### DISCUSSION THREAD

**Discussion by nikiFoe** [FreeOpcUa/python-opcua#1201] [Q&A]
Hey community,
I working on my own server, because I have more than one client connected with the server but both clients are subscribed partially to the same nodes with different sampling/publishing intervall  I am courious wheather it is possible use `set_value` refered to a specific subscription. On my current server I only take the longest intervall from subscriptions and the others I ignore because 'set_value' does trigger an change notification on all subscriptions if having the same nodes.

**Comment by zerox1212** [comment]
Subscriptions are for updating clients when a node changes. If multiple clients are subscribed to the same node they all get updated.

If you want to update one client but not another then you need to use some other special design, but I think you should rethink what you are trying to do. At least I don't understand what you are trying to do.
